infrastructure/tooling/src/importer/util.js

```diff
@@ -47,6 +47,9 @@ exports.ALLOWED_TABLES = [
   'TaskclusterIntegrationOwners',
   'TaskclusterChecksToTasks',
   'TaskclusterCheckRuns',
+
+  // (actually a continer, but that's OK)
+  'Roles'
```

Comment:
Putting it here will call `importTable` which will try to import it as an azure table in addition to `importRoles`. Was that intended?

Reply:
Oh, no, it wasn't -- I will add a special case for that.

Reply:
It looks like the loop that creates tasks is over a variable created from the `tables` properties in `services/*/azure.yml`.  So I think this is actually OK.